cmd/ptp4u/main.go

```diff
@@ -86,6 +88,10 @@ func main() {
 		log.Fatalf("Unsupported DSCP value %v", c.DSCP)
 	}
 
+	if c.DomainNumber > math.MaxUint8 {
```

Comment:
let's try to avoid using `math` and make check with constant like it's done for DSCP value

Reply:
I agree it has to be consistent in the code. Please pick one.
But please check that domain number > 0. https://go.dev/play/p/GDfLZpD83Db

Reply:
I think it will not parse negative values. and it's meaningless to compare unsigned integer to be below 0